src/index.njk

```diff
@@ -31,12 +31,23 @@ sitemapPriority: 1.0
 </div>
 
 <!-- Secondary Content -->
-<div class="w-full ff-bg-light px-6 pt-12 pb-48 sm:pt-32 sm:px-0">
+<div class="w-full ff-bg-light px-6 pt-12 pb-48 sm:pt-32">
     <div class="container m-auto text-center max-w-4xl">
         <div class="m-auto mt-36 mb-12 sm:mt-64 sm:mb-16 w-full text-center">
             <h2>Why use FlowForge for Node-RED</h2>
             <h5><a href="/node-red">What is Node-RED?</a></h5>
         </div>
+        <div class="m-auto mb-12 sm:mb-16 w-full text-center flex flex-col md:flex-row">
+            <p class="md:w-1/6 md:text-left opacity-75">Trusted by:</p>
+            <ul class="flex flex-row justify-center md:justify-between flex-wrap sm:flex-nowrap gap-[8%] w-full items-center -mb-6 md:mb-0">
+                <li class="w-[60px] pb-6 md:pb-0">{% image "./images/home-logos/heatweb-logo.png", "Heatweb logo", [60] %}</li>
+                <li class="w-[90px] pb-6 md:pb-0">{% image "./images/home-logos/enexis-logo.png", "Enexis logo", [90] %}</li>
+                <li class="w-[135px] pb-6 md:pb-0">{% image "./images/home-logos/airship-logo.png", "Airship logo", [135] %}</li>
+                <li class="w-[140px] pb-6 md:pb-0">{% image "./images/home-logos/iniationware-logo.png", "Iniationware logo", [140] %}</li>
+                <li class="w-[100px] pb-6 md:pb-0">{% image "./images/home-logos/bosch-logo.png", "Bosch logo", [100] %}
```

Comment:
Can we re-oder?

From left to right: heatweb, airship, bosch, enexis, iniationware